text/stacks/0003-stack-descriptor.md

```diff
@@ -0,0 +1,85 @@
+# Stack Descriptor and Tooling
+
+## Summary
+
+The stack-building codebase,
+[`create-stack`](https://github.com/paketo-buildpacks/stacks/tree/main/create-stack)
+should be moved into the `jam` tooling CLI, and allow users to configure their
```

Comment:
Possibly a larger question, but it seems like there's a desire to centralize around `jam`, should we maybe consider calling it something else? Like the `paketo` cli, and make a push to have it be the tool for all things Paketo?

Reply:
I would not be opposed to such a move. I'd be curious to hear what that might mean for the toolchain used by the Java buildpacks.

Reply:
Long term, I'd hope we can deprecate some things. It'll probably take a lot of work/time, but I think from a project standpoint it would be really nice to have one way to do this stuff. A good long-term goal to shoot for.